tools/duplicate_function_eliminator.py

```diff
@@ -0,0 +1,428 @@
+
```

Comment:
I'm not even clear on why you need the Minifier in this file at all? If this only does DFE, why does it need to minify?